solr/licenses/onnxruntime-LICENSE-MIT.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Looking up https://github.com/microsoft/onnxruntime/blob/v1.15.0/LICENSE here I noticed there's a 1.15.1 and 1.16.x now too, wondering about 1.15.0 vs. the later ones then.

Reply:
part-answering own question: OpenNLP 2.3.1 uses onnxruntime 1.15.0 - https://github.com/apache/opennlp/blame/opennlp-2.3.1/pom.xml#L176

Reply:
I *think* we don't specify the version of onnx, so maybe we poke them to update?

Reply:
Next opennlp will have 1.16.3 (or higher)